modyn/config/examples/modyn_config.yaml

```diff
@@ -83,6 +83,7 @@ trainer_server:
   hostname: "trainer_server"
   port: "50057"
   ftp_port: "50058"
+  offline_dataset_directory: "/tmp/offline_dataset"
```

Comment:
Please also add this to the schema :)

Reply:
OK, I added it to the schema. Also, the schema was a bit out-of-date (DSL as name, missing parameters..) and wrong (properties were not identified correctly), so I made a few more changes

Reply:
thanks!